dev/reference/transactions/transaction-isolation.md

```diff
@@ -25,7 +25,7 @@ TiDB implements Snapshot Isolation consistency, which it advertises as `REPEATAB
 
 TiDB uses the [Percolator transaction model](https://research.google.com/pubs/pub36726.html). A global read timestamp is obtained when the transaction is started, and a global commit timestamp is obtained when the transaction is committed. The execution order of transactions is confirmed based on the timestamps. To know more about the implementation of TiDB transaction model, see [MVCC in TiKV](https://pingcap.com/blog/2016-11-17-mvcc-in-tikv/).
 
-## Repeatable Read
+## Repeatable Read isolation level
```

Comment:
```suggestion
## Repeatable Read Isolation Level
```
How about this?

Reply:
Thanks for pointing out!
I have some doubts about the capitalization of headings though. I checked the related chapter in [Google developer documentation style guide](https://developers.google.cn/style/capitalization.html), and it suggests only capitalizing the first word unless it's a proper noun.
Could you please kindly tell me is "isolation level" here a proper noun? 
If so, I will commit the suggestion.

Reply:
I refer to https://www.postgresql.org/docs/current/transaction-iso.html. And the all-caps may also be OK https://dev.mysql.com/doc/refman/8.0/en/innodb-transaction-isolation-levels.html#isolevel_repeatable-read

Reply:
Thanks for the references! 
I'll commit the suggestion.

Reply:
@ran-huang Actually we follow this style: use capital letters for document titles and sentence-style for page headings. 😀You can take a look at our documentation.

Reply:
@lilin90 Thanks for pointing out.
I am still a little confused though. I noticed that some of our docs use capital letters for second-level and third-level headings, e.g. [docs/README.md](https://github.com/pingcap/docs/blob/master/README.md), while others use sentence-style, e.g. [docs/data-migration-with-binary.md](https://github.com/pingcap/docs/blob/master/dev/how-to/deploy/data-migration-with-binary.md). 
Which style should I choose for this PR?

Reply:
@ran-huang This is because almost all lines in `README.md` are document titles.

Reply:
@lilin90 Got it! Thank you for explaining in detail.😊